src/main/java/com/coursemngsys/coursemanagementsystem/hibernateControllers/CourseHibernateController.java

```diff
@@ -0,0 +1,116 @@
+package com.coursemngsys.coursemanagementsystem.hibernateControllers;
+
+import com.coursemngsys.coursemanagementsystem.Model.Course;
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.Transaction;
+
+import javax.persistence.Query;
+import javax.persistence.criteria.CriteriaQuery;
+import java.util.List;
+
+public class CourseHibernateController {
+    private SessionFactory factory = null;
+
+    public CourseHibernateController(SessionFactory factory) {
+        this.factory = factory;
+    }
+
+    private Session getSession() {
+        return factory.openSession();
+    }
+
+    public void createCourse(Course course){
+        Session session = null;
+        try{
+            session = getSession();
+            Transaction tx = session.beginTransaction();

```

Comment:
Naudojamas neaiškus kintamojo pavadinimas
```suggestion
            Transaction transaction = session.beginTransaction();

```